android/Application/src/main/java/de/tudarmstadt/es/able/PermissionClass.java

```diff
@@ -0,0 +1,86 @@
+package de.tudarmstadt.es.able;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Context;
+import android.os.Build;
+import android.provider.Settings;
+import android.support.v4.content.ContextCompat;
+import android.text.TextUtils;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Map;
+
+/**
+ * Created by user on 14.02.18.
+ */
+
+public class PermissionClass {
```

Comment:
This is a utility class, right?

I would call it "PermissionUtils" or so...

Reply:
done